org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/concurrent/RequestManagerTest.xtend

```
@@ -42,6 +46,35 @@ class RequestManagerTest {
 		sharedState = null
 	}
 
+	@Test(timeout = 1000)
+	def void testLogException() {
+		val logResult = LoggingTester.captureLogging(Level.ALL, RequestManager, [
+			val future = requestManager.runWrite([
+				throw new RuntimeException();
+			], [])
+			
+			// join future to assert log later
+			try {
+				future.get
+			} catch (Exception e) {}
```

We should probably propagate the exception here. Or use `future.join`